src/develop/pixelpipe_hb.c

```diff
@@ -1273,18 +1277,17 @@ static int pixelpipe_process_on_CPU(dt_dev_pixelpipe_t *pipe,
   }
 
   if(dt_atomic_get_int(&pipe->shutdown))
-    return 1;
+    return TRUE;
 
   /* process blending on CPU */
   dt_develop_blend_process(module, piece, input, *output, roi_in, roi_out);
   *pixelpipe_flow |= (PIXELPIPE_FLOW_BLENDED_ON_CPU);
   *pixelpipe_flow &= ~(PIXELPIPE_FLOW_BLENDED_ON_GPU);
 
   if(dt_atomic_get_int(&pipe->shutdown))
-  {
-    return 1;
-  }
-  return 0; //no errors
+    return TRUE;
+
```

Comment:
I would use a else clause here to make it clear that there is only 2 branches ending the function.

Reply:
right